e2e/e2etests/e2etests.go

```diff
@@ -997,4 +999,16 @@ var AllE2ETests = []runner.E2ETest{
 		[]runner.ArgDefinition{},
 		TestPrecompilesBankThroughContract,
 	),
+	runner.NewE2ETest(
+		TestPrecompilesDistributeName,
+		"test stateful precompiled contracts distribute",
```

Comment:
That's correct, the name is to differentiate the tests instead of creating a var like TestPrecompilesStakingDistribute...

Reply:
I think it can be a bit misleading if we adopt a convention `TestPrecompile<PrecompileName>`

But I think it's fine for now we can also refactor in the future